Codeherder19/Brians-Travel-Tracker
# Feature/final project review and tweaking

### What does this PR do?
- This PR wraps up the project aside from the readme. It includes the following actions:
1. Adds the necessary media query break points for the login page and main page.
2. Adds a normalize.scss file to improve performance across multiple browsers.
3. Ran a linter to remove unused code and unnecessary punctuation.

### What does it fix?
- Nothing to fix.

### Is this a feature or a fix?
- This is a feature.

### Where should the reviewer start?
- - Clone this repo down to your machine. CD into this file. Click on the local server link and follow the instructions in the repo to get the travel-tracker api up and running in a separate terminal. Then run npm start while in the travel tracker file and navigate to localhost:8080.

### How should this be tested?
- Log in to the app with 'username' followed by a number from 1-50 (for example username12 or username50) where the number is the id of the traveler in the traveler API. Use the password traveler2020 regardless of user name. Select a trip start date, duration, and number of travelers, and select a destination by clicking the RESERVE button on the destination card. You can then click the calculate cost button to see a preview of the estimated total, and if you click submit request, a trip card will be added to to the My Trips section with a status of 'PENDING.'
